create-event/src/reducers/user.js

```diff
@@ -0,0 +1,31 @@
+/* @flow */
+
+import * as types from '../constants/ActionTypes'
+
+const initialState = {
+  attributes: {},
+  id: null,
+  apiKey: 'sk_live_210eb57e6b95e5143c492a219091c4e5',
```

Comment:
@tmirmota You're okay with using your actual private key here too?

Reply:
It's not an admin key so I thought it would be easier for people to play around with. Easy to remove.

Reply:
@tmirmota If you're comfortable with it, so am I though I'd suggest that the placeholder just be a pattern placeholder so they know what the key should look like - `sk_live_xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`, and to let them use their own key. Thoughts on that?

Reply:
Great idea